test/lit/passes/optimize-instructions-gc.wast

```diff
@@ -2428,7 +2428,7 @@
   ;; CHECK-NEXT:   )
   ;; CHECK-NEXT:  )
   ;; CHECK-NEXT: )
-  ;; NOMNL:      (func $ref-cast-static-fallthrough-remaining (param $x eqref)
+  ;; NOMNL:      (func $ref-cast-static-fallthrough-remaining (type $eqref_=>_none) (param $x eqref)
```

Comment:
Are the changes in this file related to the PR?

Reply:
No, I think that's what I mentioned at the end of the description, `Also updates some function types in test output - I suspect that is from a race in landing of PRs.` So this fixes breakage on main. (Could be a separate PR, but the benefit to that seems slim.)